evalml/pipelines/components/component_base.py

```diff
@@ -102,6 +103,21 @@ def default_parameters(cls):
     def _supported_by_list_API(cls):
         return not cls.modifies_target
 
+    def _handle_partial_dependence_fast_mode(self, X, pipeline_parameters):
+        """Determines whether or not a component can be used with partial dependence's fast mode.
+
+        Args:
+            X (pd.DataFrame): Holdout data being used for partial dependence calculations.
+            pipeline_parameters (dict): Pipeline parameters that will be used to create the pipelines
+                used in partial dependence fast mode.
+        """
+        if self._can_be_used_for_fast_partial_dependence:
+            return pipeline_parameters
```

Comment:
do we need `pipeline_parameters` here? It seems like we can just check `_can_be_used_for_fast_partial_dependence` in `_can_be_used_for_fast_partial_dependence` and raise there error there as well. 

```
for component in pipeline.component_graph.component_instances.values():
        if not component._can_be_used_for_fast_partial_dependence:
                   raise error
```

Reply:
nvm I see what you're doing here!